cometd-nodejs-client.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface Proxy {
```

Comment:
Can you please add the license boilerplate at the beginning of the file - copy it from `cometd-nodejs-client.js`.